modules/system/controllers/updates/_list_manage_toolbar.htm

```diff
@@ -1,52 +1,94 @@
-<div data-control="toolbar">
+<div id="plugin-toolbar">
+    <div data-control="toolbar">
     <a href="<?= Backend::url('system/updates') ?>" class="btn btn-default oc-icon-chevron-left">
         <?= e(trans('system::lang.updates.return_link')) ?>
     </a>
-    <div class="btn-group">
+
+    <div class="btn-group dropdown dropdown-fixed">
         <button
-            class="btn btn-default oc-icon-magic"
-            disabled="disabled"
-            onclick="$(this).data('request-data', {
-                checked: $('.control-list').listWidget('getChecked')
-            })"
-            data-control="popup"
-            data-handler="onLoadDisableForm"
+            data-primary-button
+            type="button"
+            class="btn btn-default oc-icon-caret-down dropdown-toggle"
+            data-toggle="dropdown"
             data-trigger-action="enable"
-            data-trigger=".control-list input[type=checkbox]"
+            data-trigger=".control-list .list-checkbox input[type=checkbox]"
             data-trigger-condition="checked"
-            data-stripe-load-indicator>
-            <?= e(trans('system::lang.plugins.enable_or_disable')) ?>
+            data-request-success="$(this).prop('disabled', true).next().prop('disabled', true)">
+            <?= e(trans('system::lang.plugins.select_label')) ?>
         </button>
-        <!--
-        <button
-            class="btn btn-default oc-icon-refresh"
-            disabled="disabled"
-            onclick="$(this).data('request-data', {
-                checked: $('.control-list').listWidget('getChecked')
-            })"
-            data-request="onRefreshPlugins"
-            data-request-confirm="<?= e(trans('system::lang.plugins.refresh_confirm')) ?>"
-            data-trigger-action="enable"
-            data-trigger=".control-list input[type=checkbox]"
-            data-trigger-condition="checked"
-            data-stripe-load-indicator>
-            <?= e(trans('system::lang.plugins.refresh')) ?>
-        </button>
-        -->
+
+        <ul class="dropdown-menu" data-dropdown-title="<?= e(trans('rainlab.user::lang.users.bulk_actions')) ?>">
```

Comment:
This translation message identifier is invalid.

Reply:
@adrenth could you make a PR to fix that?